tool/flutter_tests.sh

```diff
@@ -0,0 +1,15 @@
+# Script to execute smoke tests for the flutter/tests registry
+# https://github.com/flutter/tests
+# This is executed as a pre-submit check for every PR in flutter/flutter
+
+# Generate mocks for tests
+# flutter/tests does not allow output from execution.
+./tool/generate_code.sh >> output.txt
+
+# Test devtools_shared
+cd packages/devtools_shared
+flutter test test/
+
+# Test devtools_app
+cd ../devtools_app
+flutter test test/
```

Comment:
should our integration tests be run too?

Reply:
They will need more work to prepare them for flutter/tests. I cannot get them to pass currently under the test conditions of flutter/tests. I am not familiar with devtools integration tests so I cannot change them to work with flutter/tests right now.

I think this is a really good start to prevent us from breaking devtools in flutter/flutter, it should catch the majority of changes that would affect devtools.

Reply:
Agreed, this should catch most breakages. Thanks, Kate!